api/v1/controllers/aspects.js

```diff
@@ -18,6 +18,8 @@ const doDeleteAllAssoc =
                     require('../helpers/verbs/doDeleteAllBToMAssoc');
 const doDeleteOneAssoc =
                     require('../helpers/verbs/doDeleteOneBToMAssoc');
+const doPostAssoc =
+                    require('../helpers/verbs/doPostBtoMAssoc');
 const doFind = require('../helpers/verbs/doFind');
```

Comment:
@iamigo @pallavi2209 @annyhe  Any pointers on why Travis is not able to find the file included in line 22? I have checked the file in and the tests pass locally but fail in Travis.